ACE/bin/PerlACE/Process_Unix.pm

```diff
@@ -573,16 +573,17 @@ sub Spawn ()
     return 0;
 }
 
-sub WaitKill ($)
+sub WaitKill ($;$)
```

Comment:
Maybe add some documentation about the signature, to these and other changed operations?

Reply:
I've added comments for these functions.